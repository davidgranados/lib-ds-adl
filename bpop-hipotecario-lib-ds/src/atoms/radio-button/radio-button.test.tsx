import React from "react";
import { cleanup, render } from "@testing-library/react";
import RadioButton from "./radio-button";

afterEach(cleanup);

test("RadioButton should renders ok", () => {
  const x = render(<RadioButton id="id" label="Hi" disabled={false} onChange={() => null} />);
  expect(x.container).toBeInTheDocument();
});
