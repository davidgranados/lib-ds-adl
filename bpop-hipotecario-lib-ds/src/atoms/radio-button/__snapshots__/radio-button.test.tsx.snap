// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RadioButton /> should match snapshot 1`] = `
<label
  className="wrapper wrapper--enabled"
  data-testid="radio-button-label-wrapper"
  id="radio-button-label-wrapper"
>
  <span
    className="control-wrapper"
  >
    <input
      className="input"
      data-testid="radio-button"
      id="radio-button"
      onChange={[Function]}
      type="radio"
    />
    <span
      className="control"
    >
      <span
        className="control__circle"
      />
    </span>
  </span>
  <span
    className="label"
  >
    radio-button
  </span>
</label>
`;
